static/js/actions/email_test.js

```diff
@@ -0,0 +1,47 @@
+// @flow
+import { assert } from 'chai';
+
+import {
+  actionCreatorGenerator,
+  startEmailEdit,
+  updateEmailEdit,
+  clearEmailEdit,
+
+  START_EMAIL_EDIT,
+  UPDATE_EMAIL_EDIT,
+  CLEAR_EMAIL_EDIT,
+} from './email';
+
+describe('email actions', () => {
+  describe('actionCreatorGenerator', () => {
+    it('should return a function', () => {
+      assert.isFunction(actionCreatorGenerator(START_EMAIL_EDIT));
+    });
+
+    it('returned function should be a simple action creator with no args', () => {
+      let creator = actionCreatorGenerator(START_EMAIL_EDIT);
+      assert.deepEqual({ type: START_EMAIL_EDIT }, creator());
+    });
+
+    it('returned function should be an action creator with payload with args', () => {
+      let creator = actionCreatorGenerator(START_EMAIL_EDIT);
+      let expected = { type: START_EMAIL_EDIT, payload: "HI" };
+      assert.deepEqual(expected, creator('HI'));
+    });
+  });
+
+  describe('created action helpers', () => {
```

Comment:
it doesn't seem like we're actually testing these 3 actions for anything apart from the fact that they're generic actions created by this helper method (which you've already tested above). if that's all we want to assert, maybe we should write something like this:

```javascript
const assertGenericAction(func, type_str) {
  assert.deepEqual(func(), {type: type_str})
  assert.deepEqual(func({a: 1, b: 2}), {type: type_str, payload: {a: 1, b: 2}})
}

describe('email actions', () => {
  it('should be generic', () => {
      assertGenericAction(startEmailEdit, START_EMAIL_EDIT)
      assertGenericAction(updateEmailEdit, UPDATE_EMAIL_EDIT)
      assertGenericAction(clearEmailEdit, CLEAR_EMAIL_EDIT)
  });
});
```